### PR TITLE
update task ingredient amounts

### PR DESCRIPTION
**Pull request by mayanb**

### CODE REVIEW

File: ics/v9/serializers.py [hunk 1]
```diff
@@ -172,14 +172,16 @@ def create(self, validated_data):
...
+		if matching_task_ings.count() == 0:
 			ing_query = Ingredient.objects.filter(product_type=input_creating_product, process_type=input_creating_process, recipe=None)
 			if(ing_query.count() == 0):
 				new_ing = Ingredient.objects.create(recipe=None, product_type=input_creating_product, process_type=input_creating_process, amount=0)
 			else:
 				new_ing = ing_query[0]
-			TaskIngredient.objects.create(ingredient=new_ing, task=new_input.task, actual_amount=input_creating_process.default_amount)
+			TaskIngredient.objects.create(ingredient=new_ing, task=new_input.task, actual_amount=new_input.input_item.amount)
+		else:
+			matching_task_ings.update(actual_amount=F('actual_amount') + new_input.input_item.amount)
```
Comment (sguichard) [inline ics/v9/serializers.py]: In the case where there's a recipe, should we just set the TaskIngredient actual_amount to the scaled_amount?

Comment (ishitap) [reply ics/v9/serializers.py]: Yes I thought that was the plan

Comment (ishitap) [reply ics/v9/serializers.py]: So you only need to edit it when u do something different

**Review by ishitap — CHANGES_REQUESTED** [review]
plz comment bc complicated

**Review by ishitap — APPROVED** [review]
It's fine for now, but we should put constraints in places and enforce things like only 1 ingredient for a particular proc/prod/recipe etc.

**Pull request merged**